fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1858,7 +1878,10 @@ ACTOR Future<Void> commitProxyServerCore(CommitProxyInterface proxy,
 
 	commitData.resolvers = commitData.db->get().resolvers;
 	ASSERT(commitData.resolvers.size() != 0);
-
+	for (int i = 0; i < commitData.resolvers.size(); ++i) {
+		commitData.stats.resolverDist.push_back(Histogram::getHistogram(
+		    LiteralStringRef("CommitProxy"), format("ToResolver_%d", i), Histogram::Unit::microseconds));
```

Comment:
Instead of using index, can you see if we can use resolver ID here?

Reply:
Done.